x-pack/plugins/licensing/server/plugin.ts

```diff
@@ -128,7 +128,10 @@ export class LicensingPlugin implements Plugin<LicensingPluginSetup, LicensingPl
       pollingFrequency.asMilliseconds()
     );
 
-    core.http.registerRouteHandlerContext('licensing', createRouteHandlerContext(license$));
+    core.http.registerRouteHandlerContext(
+      'licensing',
+      createRouteHandlerContext(license$, this.featureUsage.start())
```

Comment:
not ideal that we're calling `start` here during `setup`, however it doesn't take any dependencies and calling it multiple times has no effect. Ideally, we'd expose the `registerRouteHandlerContext` API on `HttpServiceStart`

Reply:
This is something @restrry was planning to do I believe.

For now, even if calling `featureUsage.start` (currently) has no side-effect, it may be better to pass down `getStartServices` instead and retrieve the start contract of the plugin that way (as we now have it as a third element of `getStartServices` return)?

Reply:
Sounds like a better idea 👍